data/json/scenarios/scenarios.json

```diff
@@ -168,5 +168,14 @@
     "allowed_locs" : ["field"],
     "map_special": "mx_helicopter",
     "flags": ["HELI_CRASH"]
-    }
+  },{
+    "type": "scenario",
+    "ident": "Mine_bottom",
+    "name": "Bottom Of A Mine",
+    "points" : 1,
+    "description": "You were mining along when you found...something. You're not sure what, but it sure is dark down here.",
```

Comment:
Just thinking thematics here, wouldn't it make more sense for it to be a
spelunker?  They'd have a lot of the same gear (probably minus a pickaxe,
but that's not super useful by itself anyway).